src/containers/Profile/ProfileDetails.js

```diff
@@ -105,10 +176,15 @@ const ProfileDetailsComponent = props => {
     body = <Loader color="blue" size="medium" />;
   }
 
-  return <div className={`col-xs-12 fullHeight ${bodyClass}`}>{body}</div>;
+  return (
+    <div>
+      {showBanner && percentageComplete < 100 && pageBanner}
+      <div className={`col-xs-12 fullHeight ${bodyClass}`}>{body}</div>
```

Comment:
this righ here is should be how you do all the styling -- by passing in a class (usually from the styles.scss you import